mitodl/open-discussions
# some dependency upgrades

#### Pre-Flight checklist


#### What are the relevant tickets?

none, just some dep upgrades

#### What's this PR do?

upgrades a bunch of packages, fixes some tests that were broken (mainly by `sinon.assert` now using a deep equality), and removes Searchkit.

#### How should this be manually tested?

everything should work, tests should pass. note that b/c of a circular dependency issue I had to run a dependency flattener - so the delta in `yarn.lock` is a little bigger than it would have been otherwise. because of this though it's probably a good idea to do some substantial checking to make sure that everything is good.
